### PR TITLE
Add org.gnome.Connections

### PR DESCRIPTION
**Pull request by felipeborges**
https://gitlab.gnome.org/felipeborges/gnome-connections

GNOME Connections is a brand new app that is a VNC and RDP client. It is based on GNOME Boxes and is part of a long-term plan of splitting off remote connection functionalities from GNOME Boxes into a dedicated application.

### CODE REVIEW

**Comment by LeandroStanger** [comment]
bot, build org.gnome.Connections

File: org.gnome.Connections.json [hunk 1]
```diff
@@ -0,0 +1,80 @@
+{
+    "app-id": "org.gnome.Connections",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "master",
```
Comment (LeandroStanger) [inline org.gnome.Connections.json:4]:
```suggestion
    "runtime-version": "3.36",
```

File: org.gnome.Connections.json [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+            ]
+        },
+        {
+            "name": "gnome-connections",
+            "builddir": true,
+            "buildsystem": "meson",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://gitlab.gnome.org/felipeborges/gnome-connections.git"
```
Comment (LeandroStanger) [inline org.gnome.Connections.json:75]:
```suggestion
                    "url": "https://gitlab.gnome.org/felipeborges/gnome-connections.git"
                    commit: 4bb705da122db60219638d5495e5d1a8750338a7
```

**Comment by felipeborges** [comment]
bot, build org.gnome.Connections

**Comment by felipeborges** [comment]
@LeandroStanger Thanks a lot for the review!

File: org.gnome.Connections.json [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+        {
+            "name" : "gtk-frdp",
+            "config-opts" : [
+                "--libdir=/app/lib"
+            ],
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://gitlab.gnome.org/gnome/gtk-frdp.git"
```
Comment (bilelmoussaoui) [inline org.gnome.Connections.json:64]: this needs to be tagged/branched as well

File: org.gnome.Connections.json [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+        },
+        {
+            "name": "gnome-connections",
+            "builddir": true,
+            "buildsystem": "meson",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://gitlab.gnome.org/felipeborges/gnome-connections.git",
+					  "tag": "v3.37.1"
```
Comment (bilelmoussaoui) [inline org.gnome.Connections.json:76]: there's some weird spacing here

Comment (felipeborges) [reply org.gnome.Connections.json]: heh, tabs vs spaces. Done.

**Comment by felipeborges** [comment]
bot, build org.gnome.Connections

**Comment by felipeborges** [comment]
@bilelmoussaoui Thank you!

File: org.gnome.Connections.json [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=RelWithDebInfo",
+                "-DWITH_OPENH264=ON",
+                "-DCMAKE_INSTALL_PREFIX=/app",
+                "-DCMAKE_INSTALL_LIBDIR=lib"
+            ],
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://pub.freerdp.com/releases/freerdp-2.0.0-rc4.tar.gz",
```
Comment (nedrichards) [inline org.gnome.Connections.json:50]: there's a more recent update just released for freerdp, does that work with this as well?

**Comment by nedrichards** [comment]
This is at 3.37.1 which is usually an unstable version, is that so? Given that you're an existing app developer with a long track record we could merge this directly to the beta branch and promote to stable when it gets a full release.

Or this could be a ruse (given that this is code that's battle tested from boxes)

**Comment by barthalion** [comment]
Yeah, I guess that as long as @felipeborges commits to maintain development release in the stable branch, we're good with that too.

File: org.gnome.Connections.json (hunk 5 above)
Comment (felipeborges) [reply org.gnome.Connections.json:50]: They often brake API/ABI, so I need a couple of days to test the new version.

**Comment by felipeborges** [comment]
@nedrichards yes, that's mostly true. To be fair, another motivation for the split of this out of Boxes was that there were not many users for this specific code path. So yes, it's been battle tested, but not that much. :)

I could have done a 3.36 release firstly. My intention is to follow the GNOME release cycle, so I picked the first public release to match the dates of the GNOME ones.

@barthalion works for me.

Thanks!

**Pull request closed without merging**